### DISCUSSION TITLE
User "Mahe's" Breeze and Breeze-Dark themes

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2335] [Ideas]
I found <a href=https://store.kde.org/p/1931624>these</a> on the KDE Store site and...wow! These might deprecate/replace the default "KDE Plasma" scheme, no? Should we get in contact with him and ask if he'd like his themes included? :)

![light-start](https://user-images.githubusercontent.com/67122280/202834243-08abc6ed-9613-4df7-ae75-bad45f599884.png)
![dark-start](https://user-images.githubusercontent.com/67122280/202834246-9bb5c09e-1e31-489a-bec1-e2c2faa179f8.png)

**Comment by ghost** [comment]
Includes dark and light variants. Very nice. I worked on the KDE-Plasma (formerly kde-plasma) theme and included orientations for the Left/Right/Top/Bottom placements. Sadly it looks like this (correctly named) Breeze theme doesn't have those, but most people use the bottom placement anyways. Also a fun thing is that this theme was uploaded on 2022-11-02.

**Reply by M4he** [reply]
Hi. I just discovered your comment. Could elaborate on the additions you included for the orientations? Looking at the current [KDE-Plasma theme's code](https://github.com/lxqt/lxqt-themes/tree/master/themes/KDE-Plasma) I'm unable to identify the related code. You are referring to the panel placement right? I don't see any difference in behavior between my Breeze themes and the existing one for different panel placements on LXQt 1.2.0.

**Reply by edwardp3** [reply]
@M4he Are the colors that appear, dependent on the Linux distro? I am running Debian sid/unstable (LXQt 1.2.0) and the tab highlights in the LXQt panel with both your dark and light Breeze themes, are blue and gray. The images above have blue and orange highlights. Thanks.

![dark-breeze](https://user-images.githubusercontent.com/73129450/215338230-cfe44a5c-2c18-446b-965f-006931580413.png)
![light-breeze](https://user-images.githubusercontent.com/73129450/215338380-05fc3120-7458-451b-9653-222451f7735e.png)

**Reply by M4he** [reply]
@edwardp3 Hi! These colors are static colors copied from Breeze itself, they don't change. The orange you see on the screenshots is the "urgent" highlight when a window calls your attention (same as Breeze). Colors on my theme are easily customizable though: you can build custom recolors on your own, see the [Creating a new color scheme](https://gitlab.com/M4he/lxqt-theme-breeze#creating-a-new-color-scheme) instructions. You'll want to change `accent-color-low`, `accent-color-medium` and `accent-color-full` at least, maybe also the `urgent-*` ones.

**Reply by edwardp3** [reply]
@M4he Understood. Thanks.  :)

**Reply by edwardp3** [reply]
@AzumaHazuki +1 for including these Breeze themes with lxqt-themes.
